README.md

```diff
@@ -42,12 +42,24 @@ specifies its file inputs and output and also its parameters. There is little re
 rule should be able to be reasoned with on its own.
 
 
-### fauna / RethinkDB credentials
+### GenBank vs Example Dataset
```

Comment:
Minor: What if you split this section into "Using GenBank data" and "Using example data"? I was initially confused by the use of "vs." here.

Reply:
Good point, will split. Or let me know if you prefer using "Input Data" similar to the Zika README

* https://github.com/nextstrain/zika#input-data

Reply:
Done, although I am tempted to change "Using GenBank data" to "Using provisioned data" to include any public data source. I am happy with "Using GenBank data" for now.